data/src/main/java/com/linkedin/data/schema/validation/ValidateDataAgainstSchema.java

```diff
@@ -124,6 +120,7 @@ public static ValidationResult validate(DataElement element, ValidationOptions o
     private Object _fixed = null;
     private boolean _valid = true;
     private final Context _context;
+    private List<FieldToTrim> _toTrim = new ArrayList<FieldToTrim>(0);
```

Comment:
Premature optimization :)

It seemed plausible to me that the most common number of fields to trim would be 0.